app/components/card.js

```diff
@@ -0,0 +1,31 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { htmlSafe } from '@ember/template';
+import { STRONG_TEXT_REGEX } from '../constants/misc';
+
+export default class CardComponent extends Component {
+  @action
+  handleButtonClick() {
+    if (!this.args.onButtonClickEventHandler) return;
+    this.args.onButtonClickEventHandler();
+  }
+
+  get formattedDescription() {
+    if (!this.args?.description) return '';
+    return htmlSafe(
```

Comment:
Doing regex operations in `htmlSafe` is complex and hard to understand. I would assign `this.args.description.replaceAll(
        STRONG_TEXT_REGEX,
        `<strong data-test-card-description-highlighted-text>$1</strong>` to a variable and then use it in `htmlSafe`

Reply:
Hey @MehulKChaudhari I have made change, please check last commit.

Reply:
Approved.